clickfile.py

```diff
@@ -472,6 +474,13 @@ def update_contracts(branch):
         update_npm=True,
     )
 
+    if with_uniswap:
+        Path(EXTERNAL_CONTRACT_PATH / "uniswap-v3").mkdir(parents=True, exist_ok=True)
```

Comment:
why don't you use update_contracts_from_git() ?

Reply:
fixed